src/live.rs

```diff
@@ -647,6 +648,53 @@ fn extract_cpio(buf: &[u8]) -> Result<Vec<u8>> {
     }
 }
 
+#[derive(Serialize)]
+struct IsoInspectOutput {
+    header: IsoFs,
+    records: Vec<String>,
+}
+
+pub fn iso_inspect(config: &IsoInspectConfig) -> Result<()> {
+    let mut iso = IsoFs::new(&config.input)?;
```

Comment:
For consistency, maybe we should use `open_live_iso(&config.input, None)` here.  We'll need an `IsoFs::for_file()` anyway when refactoring the other commands in this file.